docs/01-Introduction.md

```diff
@@ -1,5 +1,55 @@
-# 1.0 - Introduction
+# 1.0 - Introduction 
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI?
+LORIS-MRI is the backbone of the imaging component that makes up LORIS. 
+These documents assume you have some
+knowledge regarding LORIS and a functioning installation. For information
+regarding LORIS itself, please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is responsible for the processing, visualizing, and archiving
+of uploaded MRI scans. It expects an uploaded, compressed file containing
+several [DICOM][2] files, processes this into [MINC][3] and [NII][4]
+files, and then archives these files onto the server. Knowledge of
+these file formats can be helpful, but are not necessary for using or
+installing LORIS-MRI.
+
+LORIS-MRI allows you to easily organize and archive your MRI data and links it with 
+ corresponding behavioral data in LORIS. Scans can be viewed and quality controlled through the user's web browser, facilitating collaboration between radiologists, clinicians and researchers. 
+
+### 1.2 How does LORIS-MRI work?
+![user_story](images/user_story.png)
+First, users upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging Uploader.
```

Comment:
The back-end workflow... is there space to document here?  Or clarify that there is another route, to be discussed elsewhere.

Reply:
mouna discusses that here: https://github.com/aces/Loris-MRI/pull/214

Reply:
maybe just add a note saying the `upload` process and its variations is discussed in more details in another section (you cna point to it).